osf/models/node.py

```diff
@@ -149,7 +149,7 @@ def can_view(self, user=None, private_link=None):
             if not isinstance(private_link, basestring):
                 raise TypeError('"private_link" must be either {} or {}. Got {!r}'.format(str, PrivateLink, private_link))
 
-            qs |= self.filter(private_links__is_deleted=False, private_links__key=private_link)
+            return self.filter(private_links__is_deleted=False, private_links__key=private_link).filter(is_deleted=False)
```

Comment:
Minor: `.filter(...).filter(...)` is superfluous, they can all go in the same call. However, because querysets are lazy this doesn't incur any extra cost, so it's fine.